modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/util/Utils.java

```diff
@@ -485,6 +485,35 @@ public static String extractDataFromResponse(HttpResponse response, String key,
         return value;
     }
 
+    /**
+     * Extract data from response for management console requests.
+     *
+     * @param response HttpResponse
+     * @param key      key to determine the line to extract
+     * @param token    index of the value after splitting the line
+     * @return value extracted
+     * @throws IOException IOException
+     */
+    public static String extractDataFromResponseForManagementConsoleRequests(HttpResponse response, String key,
+                                                                             int token)
+            throws IOException {
+
+        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(response.getEntity().getContent()));
```

Comment:
shall we use try-with-resources to ensure BufferedReader gets closed?

Reply:
Fixed with c29d76010adf62fc72b80b080d9f5d4219393955